mlpipeline/mlpipeline/etl/pipeline.py

```
@@ -0,0 +1,234 @@
+            docs = self.parser.process_texts(ingredients)
+
+            for doc in docs:
+                qty = None
+                unit = None
+                name = ""
+
+                for ent in doc.ents:
+                    if ent.label_ == "QUANTITY":
+                        try:
+                            qty = convert_to_float(ent.text)
+                        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)